samples/nrf9160/nrf_cloud_mqtt_multi_service/src/connection.c

```diff
@@ -702,12 +709,22 @@ void manage_connection(void)
 	 * (Once we request connection, the modem will automatically try to reconnect whenever
 	 *  connection is lost).
 	 */
-
-	LOG_INF("Connecting to LTE network. This may take several minutes...");
+	LOG_INF("Setting up LTE...");
 	if (setup_lte()) {
-		LOG_ERR("LTE initialization failed. Continuing anyway. This may fail.");
+		LOG_ERR("Fatal: LTE setup failed");
```

Comment:
What happens if the SIM card is not present?

What happens if the modem firmware is corrupted or absent?  Not sure how to force that to check, though.